<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Jurnal</class>
 <widget class="QWidget" name="Jurnal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>0</y>
     <width>312</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>270</y>
     <width>141</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgeWorkday">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>241</width>
     <height>691</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
